### ISSUE TITLE
There is no way to choose which version of Visual studio to launch, and it defaults to 2013 which doesn't support syntax highlighting 

### ISSUE THREAD

**Issue by samholder** [scriptcs/scriptcs#1164]
When I use the convenience method :openvs "MyScript.csx" the created solution is 2013 format. This means the visual studio version selector uses 2013 to open the project if it is installed, and this does  not support syntax highlighting, resulting in a much poorer experience.

It would be nice to either default to 2015 or allow a way to choose 2015 as the version to open

**Comment by glennblock** [comment]
Thanks @samholder, working on it now.

**Comment by samholder** [comment]
@glennblock only just had time to test this. I rebuilt your master branch in VS and tried to launch scriptcs from the built directory, but when I did :`openvs "MyScript.csx" "2015" I just get 

> :openvs "MyScript.csx" "2015"
>(1,1): error CS7017: Member definition, statement, or end-of-file expected

I'm not sure if its to do with how I launched it, but I tried replacing the chocolatey version with the binaries from the VS build and get the same error, even if I leave out the "2015" bit

**Comment by glennblock** [comment]
You are getting this in the REPL?

**Comment by samholder** [comment]
yep

**Comment by glennblock** [comment]
Can you try running from a different folder than the bin folder i.e. Go to
the folder where your script lives and the run scriptcs using relative path?

Alternatively install svm and you can then use svm install -f to bind
directly to that folder.

**Comment by samholder** [comment]
yeah  I did that (I assumed that otherwise it wouldn't find myscript.csx) but with an absolute path (ie I went to my script folder, then `c:\path\to\the\bin\folder\scrptcs.exe` then `:openvs "MyScript.csx" "2015"`

I'll have another try tonight if I can make a few minutes

**Comment by glennblock** [comment]
I'll test it again as well. Thanks!

**Comment by samholder** [comment]
I tried this again today and it might be because my script uses 'dynamic'. This works ok with the chocolatey version, but with the version from your branch if I try and run from VS setting a script to be started in command line args it fails with a message about dynamic not being supported in this version of rosylin.

Not sure if thats related to the other issue though as even with a simple script that works when I run the program from VS with it as an argument creates the same issue in the repl launched from hitting F5 in VS

**Comment by glennblock** [comment]
@samholder you said you rebuilt my master branch? It should have been my dev branch. Can you try again using the latest dev branch in the scriptcs/scriptcs repo?

**Comment by samholder** [comment]
ok, I'll try and make some time to test this at some point this week. Thanks for the update.
